gcp-repo/category.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <site>
-   <feature url="../features/com.google.gcp.eclipse.suite.e45.feature" id="com.google.cloud.tools.eclipse.suite.e45.feature" version="0.1.0.qualifier">
+   <feature url="features/com.google.cloud.tools.eclipse.suite.e45.feature" id="com.google.cloud.tools.eclipse.suite.e45.feature" version="0.1.0.qualifier">
```

Comment:
Nope.  I'm not sure that Tycho even cares about the URL; I'll test that and remove them if unneeded.